src/test/java/csv/FileServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package csv;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FileServiceImplTest {
+    public static final FileServiceImpl csvFileService = new FileServiceImpl();
```

Comment:
you should initialize this field inside of method with @BeforeClass, the same for other similar cases